### ISSUE TITLE
Taxonomy term can not be used in user listing

### ISSUE THREAD

**Issue by lightbringer93** [statamic/cms#1269]
I added a taxonomy (profession) to my users. When I try to list the users on a team site, I receive an error. The error only appears, when I try to use my linked taxonomy in the fronted.
![Screenshot](https://user-images.githubusercontent.com/27002677/73138157-dd793380-405f-11ea-809e-85a92bd8ad8d.png)
![Screenshot (1)](https://user-images.githubusercontent.com/27002677/73138158-df42f700-405f-11ea-892a-2612d5032224.png)
```
<!-- Team wrapper
============================================= -->
<div class="container">
    <div class="flex flex-wrap lg:mb-14 -mx-2 lg:-mx-3 xl:py-3">
        {{ employees }}
            <!-- Employee
            ============================================= -->
            <div class="employee font-normal w-full md:1/2 lg:w-1/3 xl:w-1/4 px-2 lg:px-3 mb-6">
                <div class="relative overflow-hidden">
                    <img class="employee-img transition"
                         src="{{ glide:avatar width="360" }}"
                         srcset="{{ glide:avatar width="360" }} 1x,
                             {{ glide:avatar width="720" }} 2x"
                         alt="{{ firstname }} {{ lastname }}">
                    <div class="employee-name absolute transition-opacity p-1 mx-auto">
                        <h3 class="text-xl">{{ firstname }} {{ lastname }}</h3>
                    </div>
                    <div class="employee-desc absolute transition h-full w-3/5 top-0 left-0 border-r-3 border-primary-400">
                        <div class="employee-desc-bg transition-opacity absolute h-full w-full opacity-50 bg-primary-500 text-white py-3 px-3">
                            <h3 class="mb-6">{{ firstname }}<br/>{{ lastname }}</h3>
                            <div class="text-xl">
                                {{ profession | dump }}
                            </div>
                            <div class="text-lg">
                                <a class="hover:underline focus:underline"
                                   href="mailto:{{ email | obfuscate_email }}">
                                    {{ short_email ? short_email | obfuscate_email : email | obfuscate_email }}
                                </a>
                            </div>
                        </div>
                    </div>
                </div>
            </div>
        {{ /employees }}
    </div>
</div>
```
Example user:
```
name: MarcoN
roles:
  - editor
groups:
  - administration
lastname: Nägeli
firstname: Marco
avatar: team/marco-naegli.jpg
profession:
  - 'professions::coo'
id: 31573fe2-4e6d-4759-96c3-bb89536212c5
```

**Comment by jasonvarga** [comment]
At the moment only entries may be taxonomized.

**Comment by jasonvarga** [comment]
I believe fixed by 44d30f70, however it only suppresses the error, it doesn't actually taxonomize the user.
